diff -ru setup-2.8.56/group setup-2.8.56-droidcontainer/group
--- setup-2.8.56/group	2012-07-13 12:58:49.000000000 +0200
+++ setup-2.8.56-droidcontainer/group	2013-05-31 09:35:09.217147095 +0200
@@ -18,6 +18,6 @@
 dip::40:
 ftp::50:
 lock::54:
-audio::63:
-nobody::99:
+audio::1005:
+nobody::9999:
 users::100:
diff -ru setup-2.8.56/uidgid setup-2.8.56-droidcontainer/uidgid
--- setup-2.8.56/uidgid	2012-07-13 12:58:49.000000000 +0200
+++ setup-2.8.56-droidcontainer/uidgid	2013-05-31 09:35:34.139085596 +0200
@@ -69,7 +69,7 @@
 piranha	60	60	/etc/sysconfig/ha		/dev/null	piranha
 prelude-manager	61	61	-		/sbin/nologin	prelude-manager
 snortd	62	62	-		/sbin/nologin	snortd
-audio	-	63	-		-	setup
+audio	-	1005	-		-	setup
 condor	64	64	/var/lib/condor		/sbin/nologin	condord
 nslcd	65	(55)	/		/sbin/nologin	nslcd
 wine	-	66	-		-	wine
